Components/badge/badge.css

```diff
@@ -0,0 +1,94 @@
+.VariantContainer
+{
+      margin-top: 25px;
+      border: 3px solid var(--primary-color);
+      width: 900px;
+      height: 200px;
```

Comment:
container fixed height , width is not recommended for responsiveness use rems!!

can use rems for margin